e2e/test_operator_upgrades_variations/operator_upgrades_variations_test.go

```diff
@@ -183,7 +183,7 @@ var _ = Describe("Operator Upgrades", Label("e2e", "pr"), func() {
 	// this setup allows to dynamically generate the table entries that will be executed e.g. to test different upgrades
 	// for different versions without hard coding or having multiple flags.
 	DescribeTable(
-		"upgrading a cluster without chaos",
+		"upgrading a cluster without chaos", Label("quick"),
```

Comment:
nit: Maybe a more descriptive label than 'quick'? It is not self-describing.

Reply:
That makes sense 👍